pages/flow/index.js

```diff
@@ -70,7 +76,15 @@ export default function Flow() {
                   <p>{asana.english_name}</p>
                   <StyledAddButton
                     onClick={() => {
-                      setFlowAsanaIds([...flowAsanaIds, asana.id]);
+                      setSelectedAsanas([
+                        ...selectedAsanas,
+                        {
+                          id: asana.id,
+                          english_name: asana.english_name,
+                          img_url: asana.img_url,
+                        },
+                      ]);
```

Comment:
In a later user story I want to store that information in the local storage and I thought, it might be better if I don't store all the information there?